src/chk/check.pb-c.c

```diff
@@ -0,0 +1,281 @@
+  { "CIC_ENGINE_NONEXIST_IN_MAP", 8 },
+  { "CIC_NONE", 0 },
+  { "CIC_OBJ_DATA_INCONSIST", 20 },
+  { "CIC_OBJ_LOST_EC_DATA", 19 },
+  { "CIC_OBJ_LOST_EC_SHARD", 18 },
+  { "CIC_OBJ_LOST_REP", 17 },
+  { "CIC_POOL_BAD_LABEL", 7 },
+  { "CIC_POOL_BAD_SVCL", 6 },
+  { "CIC_POOL_LESS_SVC_WITHOUT_QUORUM", 2 },
+  { "CIC_POOL_LESS_SVC_WITH_QUORUM", 1 },
+  { "CIC_POOL_MORE_SVC", 3 },
+  { "CIC_POOL_NONEXIST_ON_ENGINE", 5 },
+  { "CIC_POOL_NONEXIST_ON_MS", 4 },
+  { "CIC_UNKNOWN", 21 },
+};
+const ProtobufCEnumDescriptor chk__check_inconsist_class__descriptor =
+{
+  PROTOBUF_C__ENUM_DESCRIPTOR_MAGIC,
+  "chk.CheckInconsistClass",
+  "CheckInconsistClass",
+  "Chk__CheckInconsistClass",
+  "chk",
+  22,
+  chk__check_inconsist_class__enum_values_by_number,
+  22,
+  chk__check_inconsist_class__enum_values_by_name,
+  2,
+  chk__check_inconsist_class__value_ranges,
+  NULL,NULL,NULL,NULL   /* reserved[1234] */
+};
+static const ProtobufCEnumValue chk__check_inconsist_action__enum_values_by_number[13] =
+{
+  { "CIA_DEFAULT", "CHK__CHECK_INCONSIST_ACTION__CIA_DEFAULT", 0 },
+  { "CIA_INTERACT", "CHK__CHECK_INCONSIST_ACTION__CIA_INTERACT", 1 },
+  { "CIA_IGNORE", "CHK__CHECK_INCONSIST_ACTION__CIA_IGNORE", 2 },
+  { "CIA_DISCARD", "CHK__CHECK_INCONSIST_ACTION__CIA_DISCARD", 3 },
+  { "CIA_READD", "CHK__CHECK_INCONSIST_ACTION__CIA_READD", 4 },
+  { "CIA_TRUST_MS", "CHK__CHECK_INCONSIST_ACTION__CIA_TRUST_MS", 5 },
+  { "CIA_TRUST_PS", "CHK__CHECK_INCONSIST_ACTION__CIA_TRUST_PS", 6 },
+  { "CIA_TRUST_TARGET", "CHK__CHECK_INCONSIST_ACTION__CIA_TRUST_TARGET", 7 },
+  { "CIA_TRUST_MAJORITY", "CHK__CHECK_INCONSIST_ACTION__CIA_TRUST_MAJORITY", 8 },
+  { "CIA_TRUST_LATEST", "CHK__CHECK_INCONSIST_ACTION__CIA_TRUST_LATEST", 9 },
+  { "CIA_TRUST_OLDEST", "CHK__CHECK_INCONSIST_ACTION__CIA_TRUST_OLDEST", 10 },
+  { "CIA_TRUST_EC_PARITY", "CHK__CHECK_INCONSIST_ACTION__CIA_TRUST_EC_PARITY", 11 },
+  { "CIA_TRUST_EC_DATA", "CHK__CHECK_INCONSIST_ACTION__CIA_TRUST_EC_DATA", 12 },
+};
+static const ProtobufCIntRange chk__check_inconsist_action__value_ranges[] = {
+{0, 0},{0, 13}
+};
+static const ProtobufCEnumValueIndex chk__check_inconsist_action__enum_values_by_name[13] =
+{
+  { "CIA_DEFAULT", 0 },
+  { "CIA_DISCARD", 3 },
+  { "CIA_IGNORE", 2 },
+  { "CIA_INTERACT", 1 },
+  { "CIA_READD", 4 },
+  { "CIA_TRUST_EC_DATA", 12 },
+  { "CIA_TRUST_EC_PARITY", 11 },
+  { "CIA_TRUST_LATEST", 9 },
+  { "CIA_TRUST_MAJORITY", 8 },
+  { "CIA_TRUST_MS", 5 },
+  { "CIA_TRUST_OLDEST", 10 },
+  { "CIA_TRUST_PS", 6 },
+  { "CIA_TRUST_TARGET", 7 },
+};
+const ProtobufCEnumDescriptor chk__check_inconsist_action__descriptor =
+{
+  PROTOBUF_C__ENUM_DESCRIPTOR_MAGIC,
+  "chk.CheckInconsistAction",
+  "CheckInconsistAction",
+  "Chk__CheckInconsistAction",
+  "chk",
+  13,
+  chk__check_inconsist_action__enum_values_by_number,
+  13,
+  chk__check_inconsist_action__enum_values_by_name,
+  1,
+  chk__check_inconsist_action__value_ranges,
+  NULL,NULL,NULL,NULL   /* reserved[1234] */
```

Comment:
(style)  space required after that ',' (ctx:VxV)
(style)  space required after that ',' (ctx:VxV)
(style)  space required after that ',' (ctx:VxV)